meteor/server/api/peripheralDevice.ts

```diff
@@ -256,12 +257,19 @@ export namespace ServerPeripheralDeviceAPI {
 							PlayoutLockFunctionPriority.CALLBACK_PLAYOUT,
 							async () => {
 								const rundownIDs = Rundowns.find({ playlistId }).map((r) => r._id)
+								const partInstanceIDs = [
+									activePlaylist.currentPartInstanceId,
+									activePlaylist.nextPartInstanceId,
```

Comment:
I suspect we can consider just the currentPartInstanceId.

next hasnt started playback, and so shouldnt have any adlib pieces which might need updating.
previous has finished playback (or just the overlap remaining) and so probably shouldnt have any left to update